apps/minifront/src/components/ibc/deposit-manual/hooks.ts

```diff
@@ -145,51 +105,30 @@ const usePenumbraIbcDenoms = () => {
     data: ibcAddrs,
     isLoading: ibcAddrsLoading,
     error: ibcAddrssErr,
-  } = useQuery(
-    ['penumbraIbcDenoms', registry],
-    async () => generatePenumbraIbcDenoms(registry?.ibcConnections ?? []),
-    {
-      enabled: Boolean(registry),
-    },
-  );
+  } = useQuery({
+    queryKey: ['penumbraIbcDenoms', registry],
+    queryFn: async () => generatePenumbraIbcDenoms(registry?.ibcConnections ?? []),
+    enabled: Boolean(registry),
+  });
 
   return {
     data: ibcAddrs,
     isLoading: registryIsLoading || ibcAddrsLoading,
-    error: registryErr || ibcAddrssErr,
+    error: registryErr ?? ibcAddrssErr,
```

Comment:
worth considering a separate effort to better capture IBC error codes and provide more user-friendly shielding / unshielding error messages

<img width="522" alt="Screenshot 2024-12-11 at 12 39 43 PM" src="https://github.com/user-attachments/assets/44d200e5-7dcc-44ca-8f92-f12034bf700a" />

<img width="1077" alt="Screenshot 2024-12-11 at 12 39 13 PM" src="https://github.com/user-attachments/assets/432f6568-ed87-490a-84c5-2c0263c97c28" />